docs/reference/env_variables.rst

```diff
@@ -0,0 +1,177 @@
+.. meta::
+    :description: HIP environment variables reference
+    :keywords: AMD, HIP, environment variables, environment, reference
+
+*************************************************************
+HIP environment variables
+*************************************************************
+
+In this section, the reader can find all the important HIP environment variables. The full collection of the ROCm environment variables is on the :doc:`ROCm environment variables page<rocm:reference/env-variables>`.
+
+GPU isolation variables
+=======================
+
+The GPU isolation environment variables in HIP are collected in the next table. For more information, check :doc:`GPU isolation page <rocm:conceptual/gpu-isolation>`.
```

Comment:
This link page has 5 environment variables while here we see only three. Why the different numbers and what do these three have that the others lack?

Reply:
``OMP_DEFAULT_DEVICE`` variable is for OpenMP runtime, ``HIP_VISIBLE_DEVICES`` and ``CUDA_VISIBLE_DEVICES`` added as one line.